### ISSUE TITLE
Shell run-main vs runMain

### ISSUE THREAD

**Issue by bgyori** [clulab/eidos#440]
The command for running the shell has been
```
sbt 'run-main org.clulab.wm.eidos.apps.EidosShell'
```
for a long time and it used to work fine. But on the latest master, I am getting this error:
```
[error] Expected ID character
[error] Not a valid command: run-main
[error] Expected project ID
[error] Expected configuration
[error] Expected ':'
[error] Expected key
[error] Not a valid key: run-main (similar: runMain, bgRunMain, run)
[error] run-main org.clulab.wm.eidos.apps.EidosShell
[error]         ^
[INFO] [09/21/2018 01:12:11.962] [Thread-8] [CoordinatedShutdown(akka://sbt-web)] Starting coordinated shutdown from JVM shutdown hook
```
when I change it to
```
sbt 'runMain org.clulab.wm.eidos.apps.EidosShell'
```
it works!

Should I PR this change?

**Comment by kwalcock** [comment]
I can get it.  sbt changed the name of it's command.  It will have to be
changed in several other places as well.

**Comment by bgyori** [comment]
Fixed in #442
